src/components-examples/material/tree/tree-nested-overview/tree-nested-overview-example.html

```diff
@@ -1,15 +1,12 @@
 <mat-tree [dataSource]="dataSource" [treeControl]="treeControl" class="example-tree">
   <!-- This is the tree node template for leaf nodes -->
   <mat-tree-node *matTreeNodeDef="let node" matTreeNodeToggle>
-    <li class="mat-tree-node">
       <!-- use a disabled button to provide padding for tree leaf -->
-      <button mat-icon-button disabled></button>
+      <button mat-icon-button disabled aria-hidden="true"></button>
```

Comment:
I'm not sure that we should be providing a button just to add some padding either.